packages/website/src/components/Prototyping.tsx

```diff
@@ -1,5 +1,5 @@
 import { type FC, useRef, useEffect, useState } from "react";
-import { type Triggers, type Trigger } from "@nlxai/journey-manager";
```

Comment:
previously, you were only importing types. When you started importing code here, server side rendering would get stuck on the web apis that were used in the journey-manager.

Another approach is exporting the icons in a separate file, but I didn't want to deal with creating another rollup config, etc.